### ISSUE TITLE
Question: key servers for public keys

### ISSUE THREAD

**Issue by JayBrown** [jpouellet/signify-osx#4]
Do key servers exist where the user can upload his public key file? Would come in handy, I guess, but I couldn't find any.

**Comment by jpouellet** [comment]
Signify is designed such that keys are short enough to be posted just about anywhere. There are not "key servers" in the pgp sense, but you can find signify keys all over the place.

More about the design and philosophy:
- http://www.openbsd.org/papers/bsdcan-signify.html
- http://www.tedunangst.com/flak/post/signify

Some keys:
- http://cvsweb.openbsd.org/cgi-bin/cvsweb/src/etc/signify/
- https://twitter.com/reykfloeter/status/462224830017044480
- https://github.com/jpouellet/signify-osx/tree/master/src/etc/signify

At one point I verified my keys against the keys of several OpenBSD devs at BSDCan, pgp-signed those, and posted them someplace, but I can't seem to find that post right now and don't have a trusted-enough computer with me right now to repeat the process in a meaningfully-trustable way.

tl;dr - Post them anywhere and everywhere, and rely on distributed mutual peer verification. There is no central signify key distribution infrastructure and likely never will be.

*These are not authoritative.* Somebody might be MITMing me right now and I'm not at a computer with a previously trusted copy, but here's another copy of some of the keys as I have them right now:
```
untrusted comment: openbsd 5.8 base public key
RWQNNZXtC/MqP3Eiu+6FBz/qrxiWQwDhd+9Yljzp62UP4KzFmmvzVk60

untrusted comment: openbsd 5.9 base public key
RWQJVNompF3pwfIqbg+5sxfpxmZMa3tTBaW4qbUhWje/H/M7glrA6oVn

untrusted comment: openbsd 6.0 base public key
RWSho3oKSqgLQy+NpIhFXZJDtkE65tzlmtC24mStf8DoJd2OPMgna4u8

untrusted comment: LibreSSL Portable public key
RWQg/nutTVqCUVUw8OhyHt9n51IC8mdQRd1b93dOyVrwtIXmMI+dtGFe
```
If you make your own, distribute them in a similar fashion, perhaps signed by other keys your intended recipients might already have trusted copies of.
